atlasdb-client/build.gradle

```diff
@@ -62,7 +62,9 @@ dependencies {
   compile group: 'com.palantir.safe-logging', name: 'safe-logging'
   compile (group: 'com.palantir.tritium', name: 'tritium-lib') {
     exclude (group: 'io.dropwizard.metrics', module: 'metrics-core')
+    exclude (group: 'org.hdrhistogram', module: 'HdrHistogram')
```

Comment:
We should probably rollback this in Atlas when they fix the double dependency issue, but there's no other way around it for now.